examples/TripPin.openapi3.json

```diff
@@ -3301,6 +3301,9 @@
                 "title": "Location",
```

Comment:
@ralfhandl Also need a bit of help here. When I run `test.cmd` I get this error. I makes no sense to me.

```
odata-rw-v3
lexical error: invalid char in json text.
                                        {"swagger":"2.0","info":{"titl
                     (right here) ------^
lexical error: invalid char in json text.
                                        {"openapi":"3.0.0","info":{"ti
                     (right here) ------^
TripPin
lexical error: invalid char in json text.
                                        {"swagger":"2.0","info":{"titl
                     (right here) ------^
lexical error: invalid char in json text.
                                        {"openapi":"3.0.0","info":{"ti
                     (right here) ------^
```
As a result the json files are empty

Reply:
@ralfhandl do you have any clue why this is happening?

Reply:
The only cause I can think of is that `test.cmd` only works if called from within the `tools` folder.

Note that the XSLT and the JS versions are not in sync and have their own set of test inputs and expected outputs.

Reply:
I have a suspisseion this has something to do with CRLF and LF line endings.

I changed `autocrlf = true` to `autocrlf = input` a while ago and this problem started happening when I merged master in.
Also the CI doesn't seem to have this problem.

Reply:
I have `autocrlf = false`

Reply:
FML..... it was caused by a missing `"` in a `xsl:text` block